### PR TITLE
Home tasks by Roman Lobodenko

### PR DESCRIPTION
**Pull request by RLobodenko**

### CODE REVIEW

File: rlobodenko/01_CSS/new1.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```
Comment (pluhin) [inline rlobodenko/01_CSS/new1.html:1]: Where is font declaration from googlefonts site?

File: rlobodenko/01_HTML/new4.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>  
+<HTML> 
+	<HEAD>
+		<META charset="utf-8">
+		<TITLE>Пример страницы</TITLE>
+	</HEAD>  
+	<BODY> 
+	<H1>Заголовок первого уровня</H1> 
+	<P>	x | o |</P>
+<P>‐‐‐‐‐‐‐‐‐‐‐</P>
```
Comment (pluhin) [inline rlobodenko/01_HTML/new4.html:17]: here pre tag

File: rlobodenko/05_HTML/new_2.html [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+</select>
+	</label>
+	</fieldset>
+	</form>
+	</fieldset>
+	
+	
+		<FIELDSET> 
+<LEGEND>Данные для входа и восстановления пароля</LEGEND> 
+<input type="text" placeholder="Логин" pattern="[A-Za-z]{5}+" placeholder="[A-Za-z]{5}+" SIZE="20"> <br><span> </span>
```
Comment (pluhin) [inline rlobodenko/05_HTML/new_2.html:100]: where is name?

File: rlobodenko/05_HTML/new_2.html [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+<input type="password" placeholder="Пароль" pattern="[A-Za-z]{10}+" placeholder="[A-Za-z]{10}+" SIZE="20"><br><span> </span>
```
Comment (pluhin) [inline rlobodenko/05_HTML/new_2.html:101]: where is name here

File: rlobodenko/05_HTML/new_1.html [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+<input type="submit" value="Загрузить"><br>
+	</fieldset>
+</form></TD>
+<TD><hr>
+<h2>Валидация данных HTML5:</h2>
+<form method="get" action="http://sc.it-academy.by/fe/form.php">
+			<FIELDSET> 
+<LEGEND>Строка поиска</LEGEND>  
+<input type="email" name="e-mail" placeholder="Только русские буквы"> <br>
+<label>Телефонный номер в формате<br>
```
Comment (pluhin) [inline rlobodenko/05_HTML/new_1.html:167]: label for?

File: rlobodenko/05_HTML/new_1.html [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+<TD><hr>
+<h2>Загрузка файла на сервер:</h2>
+<form method="post" action="http://sc.it-academy.by/fe/form.php" enctype="multipart/form-data">
+			<FIELDSET> 
+<LEGEND>Upload</LEGEND> 
+<label>Файл:</label>
+<label>
+<input type="file" name="file_Field" dir="ltr">
+</label>
+<label>Срок хранения (года):</label>
```
Comment (pluhin) [inline rlobodenko/05_HTML/new_1.html:152]: label for?

File: rlobodenko/05_HTML/new_1.html [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+<input type="submit" value="Ок"><br>
+	</fieldset>
+</form></TD></TR>
+<TR>
+<TD><hr>
+<h2>Загрузка файла на сервер:</h2>
+<form method="post" action="http://sc.it-academy.by/fe/form.php" enctype="multipart/form-data">
+			<FIELDSET> 
+<LEGEND>Upload</LEGEND> 
+<label>Файл:</label>
```
Comment (pluhin) [inline rlobodenko/05_HTML/new_1.html:148]: label for?

**Review by pluhin — CHANGES_REQUESTED** [review]

**Comment by RLobodenko** [comment]
Почему в файле 05_HTML/new_1.html в задании upload не показывает строку загрузки?

**Comment by RLobodenko** [comment]
Why in the file 05_HTML / new_1.html in the task upload does not show the download string?

**Comment by pluhin** [comment]
You have conflicts, could you resolve them

**Comment by pluhin** [comment]
> Why in the file 05_HTML / new_1.html in the task upload does not show the download string?

What do you mean "download string"? Status bar? Progress of downloading?

**Comment by RLobodenko** [comment]
..спросил этот вопрос в slack'е

**Pull request merged**